scripts/ci-release.py

```diff
@@ -0,0 +1,37 @@
+import toml
+import requests
+import subprocess
+
+cargo_toml = toml.load("Cargo.toml")
+crate_version = cargo_toml["package"]["version"]
+print("Detected crate version " + crate_version)
+
+api_url = "https://crates.io/api/v1/crates/x86_64/versions"
+crates_io_versions = requests.get(api_url).json()
```

Comment:
This can be made (slightly) more efficient by querying something like `https://crates.io/api/v1/crates/x86_64/0.14.3` directly, and then just checking if the returned JSON has a `"version"` field, or that `version.num` is the requested field.

Reply:
But the existing solution is also fine, let's see if it works for 0.14.4

Reply:
Good idea! I didn't know about this API endpoint. Is there a complete reference for it somewhere?